server/base/src/main/java/org/apache/accumulo/server/util/DumpZookeeper.java

```diff
@@ -129,7 +129,7 @@ private static void write(PrintStream out, int indent, String fmt, Object... arg
     for (int i = 0; i < indent; i++) {
       out.print("  ");
     }
-    out.println(String.format(fmt, args));
+    out.printf((fmt) + "%n", args);
```

Comment:
```suggestion
    out.printf(fmt + "%n", args);
```

Reply:
I just accepted the IDE suggested changes - looking at your suggestion, I can't find where adding the %n is necessary to add to the calling fmt string.

Reply:
Updated with proper newlines added in #76fa72c862